packaging/Dockerfile.centos_7_git_rpm

```diff
@@ -1,19 +1,9 @@
-# This Dockerfile is based on the recommendations provided in the
-# Centos official repository (https://hub.docker.com/_/centos/).
-# It enables systemd to be operational.
-FROM centos:7.5.1804
-ENV container docker
-COPY docker_enable_systemd.sh docker_sys_config.sh ./
-RUN bash ./docker_enable_systemd.sh && rm ./docker_enable_systemd.sh -f
+FROM nmstate/centos_7_base
 
-# libibverbs is a dependency for openvswitch that is missing as a dependency in the RPM
-RUN yum -y upgrade && \
-    yum -y install NetworkManager NetworkManager-ovs openvswitch libibverbs && \
-    yum -y install epel-release yum-utils && \
+RUN yum -y install yum-utils && \
```

Comment:
Isn't this suppose to be part of base?

Reply:
No, the CI system does not need to install the COPR repository.